tests/framework/framework.go

```diff
@@ -132,14 +132,14 @@ func NewFramework(prefix string, config Config) (*Framework, error) {
 	// clients
 	kcs, err := f.GetKubeClient()
 	if err != nil {
-		err = errors.Wrap(err, "ERROR, unable to create K8SClient")
+		return nil, errors.Wrap(err, "ERROR, unable to create K8SClient")
 	} else {
 		f.K8sClient = kcs
 	}
 
 	cs, err := f.GetCdiClient()
 	if err != nil {
-		err = errors.Wrap(err, "ERROR, unable to create CdiClient")
+		return nil, errors.Wrap(err, "ERROR, unable to create CdiClient")
```

Comment:
ditto

Reply:
I'm sure you're busy... my re-review today shows no code change and no comment from you. Will wait to merge.

Reply:
I was on PTO Tuesday, that is why I didn't answer.